alchemist-physics/src/test/resources/pushing_behavior.yml

```diff
@@ -27,7 +26,7 @@ deployments:
       - time-distribution:
           type: DiracComb
           parameters: [1.0]
-        type: BlendedSteeringWithPhysics
+        type: PhysicalBlendedSteering
         actions:
-          - type: CognitiveAgentSeek
+          - type: CognitiveAgentSeek2D
```

Comment:
Do we also have a 3D version?

Reply:
We do not have a 3D version. I switched to the 2D as it was documented having more sophisticated  movements.